themes/src/main/resources/theme/base/login/register.ftl

```diff
@@ -104,6 +104,18 @@
                             </span>
                         </#if>
                     </div>
+                    <div class="${properties.kcInputWrapperClass!}" id="password-entropy" data-password-input="password" style="display: none;">
+                        <div class="progress" style="margin-bottom: 0; margin-top: 1px;">
```

Comment:
If possible, I'd like to avoid in-line styling. Preferably this style would be added to a separate style sheet.